tensor2tensor/rl/envs/py_func_batch_env.py

```diff
@@ -20,12 +20,10 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
-import gym
 
 from tensor2tensor.rl.envs.in_graph_batch_env import InGraphBatchEnv
-
 import tensorflow as tf
-
+from tensor2tensor.rl.envs import utils
```

Comment:
This is a cyclic import, it must be avoided.

Reply:
Done. The above comment applies.